README.md

```diff
@@ -71,10 +73,11 @@ This [dbt](https://github.com/dbt-labs/dbt) package provides compatibility with
 | SQL generators        | star                          | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | union_relations               | :white_check_mark:    | custom macro provided                                                  |
 | SQL generators        | generate_series               | :white_check_mark:    | custom macro provided                                                  |
-| SQL generators        | surrogate_key                 | :white_check_mark:    | custom macro provided, [additional install steps required](#additional-steps-for-surrogate_key-macro)               |
+| SQL generators        | generate_surrogate_key             | :white_check_mark:    | custom macro provided, [additional install steps required](#additional-steps-for-surrogate_key-macro)               |
 | SQL generators        | safe_add                      | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | pivot                         | :white_check_mark:    | no customization needed                                                |
 | SQL generators        | unpivot                       | :white_check_mark:    | no customization needed, see [compatibility note](#unpivot)            |
+| SQL generators        | deduplicate                        | :white_check_mark:    | custom macro provided                                                |
```

Comment:
Correct indentation

Reply:
These will be converted into table, indentation won't matter much, Mohan.